### DISCUSSION TITLE
Mac M1 support?

### DISCUSSION THREAD

**Discussion by psommerfeld** [roflcoopter/viseron#689] [General]
Just found this project!
Can I run it on Mac M1?
Would the roflcoopter/aarch64-viseron work?
Does it support Mac M1's NEON SIMD acceleration?
Thanks!

**Comment by roflcoopter** [comment]
I'm not sure if it will work, would be great if you could try it out!

If ffmpeg or gstreamer supports hardware acceleration on it then Viseron can utilize that
